### ISSUE TITLE
Agent Flow Windows installer logging arguments 

### ISSUE THREAD

**Issue by timo1707** [grafana/agent#3992]
### Request

Adding the possability to define a log path argument when Install the Grafana Agend on Windows.
e.g. 
`grafana-agent-flow-installer.exe /S /log="C:\Install\Logs\grafana-agent-flow-installer.log"`

### Use case

On larger Environments are usualy specific Logging paths e.g. "C:\Install\Logs\software01.log" required to monitor installation processes. 
e.g. on a new deployment several Software packages getting automaticly installed . to debug or control if all packages are installed, the relevant log path is getting analysed.

**Comment by rfratto** [comment]
Just to clarify, we're talking about the installer logs and not logs for the agent itself, right?

**Comment by timo1707** [comment]
> Just to clarify, we're talking about the installer logs and not logs for the agent itself, right?

exactly, the logs out of the [installer Script](https://github.com/grafana/agent/blob/main/packaging/grafana-agent-flow/windows/install_script.nsis)

**Comment by jkroepke** [comment]
This seems to be possible with `LogSet on` inside the installer, however it required a non-default build of NSIS

> Note that the NSIS_CONFIG_LOG build setting must be set (scons NSIS_CONFIG_LOG=yes) when building (it is not set by default) to support this.

Ref: https://nsis.sourceforge.io/Docs/Chapter4.html#installlog

Lucky, NSIS provides Windows binaries with on SourceForce (https://nsis.sourceforge.io/Special_Builds#Advanced_logging), with Logging support, but it needs to be guaranteed that the binary is available on the CI.
